### PR TITLE
python310Packages.pyte: 0.8.0 -> 0.8.1

### PR DESCRIPTION
**Pull request by onny**
###### Things done

- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- [ ] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
- [x] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [ ] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [22.11 Release Notes (or backporting 22.05 Release notes)](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#generating-2211-release-notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
  - [ ] (Release notes changes) Ran `nixos/doc/manual/md-to-db.sh` to update generated release notes
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/development/python-modules/pyte/default.nix [hunk 1]
```diff
@@ -1,12 +1,19 @@
-{ lib, buildPythonPackage, fetchPypi, pytestCheckHook, pytest-runner, wcwidth }:
+{ lib
+, buildPythonPackage
+, fetchFromGitHub
+, pytestCheckHook
+, pytest-runner
+, wcwidth }:
```
Comment (SuperSandro2000) [inline pkgs/development/python-modules/pyte/default.nix:6]:
```suggestion
, wcwidth
}:
```

File: pkgs/development/python-modules/pyte/default.nix [hunk 2]
```diff
@@ -1,12 +1,19 @@
...
-    inherit pname version;
-    sha256 = "7e71d03e972d6f262cbe8704ff70039855f05ee6f7ad9d7129df9c977b5a88c5";
+  src = fetchFromGitHub {
+    owner = "selectel";
+    repo = pname;
+    rev = version;
+    sha256 = "sha256-gLvsW4ou6bGq9CxT6XdX+r2ViMk7z+aejemrdLwJb3M=";
   };
 
   nativeBuildInputs = [ pytest-runner ];
```
Comment (SuperSandro2000) [inline pkgs/development/python-modules/pyte/default.nix:19]:
```suggestion
```
please patch that requirement for it out. It is not working in the sandbox.

**Comment by onny** [comment]
@SuperSandro2000 Thank you for the info about `pytest-runner`, fixed it :)

**Review by jonringer — APPROVED** [review]
LGTM

Result of nixpkgs-review pr 183379 run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)<details>  <summary>9 packages built:</summary>  <ul>    <li>python310Packages.bpython</li>    <li>python310Packages.curtsies</li>    <li>python310Packages.pyte</li>    <li>python39Packages.bpython</li>    <li>python39Packages.curtsies</li>    <li>python39Packages.pyte</li>    <li>termtosvg</li>    <li>thefuck</li>    <li>xonsh</li>  </ul></details>

**Pull request merged**